mam/api/api.c

```diff
@@ -505,6 +504,7 @@ retcode_t mam_api_bundle_read(mam_api_t *const api, bundle_transactions_t const
                               size_t *const payload_size, bool *const is_last_packet) {
   retcode_t ret = RC_OK;
   trit_t tag[NUM_TRITS_TAG];
+  trit_t *const msg_id = tag;
```

Comment:
Why do we introduce this ? Just for the name ? Is that really necessary ?

Reply:
kind of annoyed me that in one place we use `msg_id` as key for ctx map and in the other we use `tag`,
if you think it's not necessary, i'll accept

Reply:
I don't have a strong opinion on this, I usually go for less variables when that's possible but I understand the idea

Reply:
I am also in favor of less, but i prefer consistency/clarity, so this time, i am ok with the cost